docs/src/configuration/services/elasticsearch.md

```diff
@@ -32,6 +32,18 @@ To switch to OpenSearch, follow the same procedure as for [upgrading](#upgrading
 Elasticsearch 7.9 isn't available in the EU-1 and US-1 regions.
 Consider [region migration](../../guides/general/region-migration.md) if your project is in those regions.
 
+## Use a framework
+
+If you use one of the frameworks, follow the guide there:
```

Comment:
As we discussed, I'd try to avoid the "there". Something like this might work (though feel free to adjust):
```suggestion
If you use one of the following frameworks, follow its guide:
```

Also, I'd note that since you're using the same language in each file, it might make sense to have it be a shortcode. The specific list could be the inner content.

It would make it easier to change the language in all of them, and it should be changed in each.